iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/impl/bicep/OutputDeclarationImpl.java

```diff
@@ -30,14 +30,15 @@
 import org.sonar.iac.arm.tree.api.StringLiteral;
 import org.sonar.iac.arm.tree.api.bicep.Decorator;
 import org.sonar.iac.arm.tree.api.bicep.HasDecorators;
+import org.sonar.iac.arm.tree.api.bicep.HasKeyword;
 import org.sonar.iac.arm.tree.api.bicep.InterpolatedString;
 import org.sonar.iac.arm.tree.api.bicep.SyntaxToken;
 import org.sonar.iac.arm.tree.impl.AbstractArmTreeImpl;
 import org.sonar.iac.common.api.tree.Tree;
 
 import static org.sonar.iac.arm.tree.ArmHelper.addChildrenIfPresent;
 
-public class OutputDeclarationImpl extends AbstractArmTreeImpl implements OutputDeclaration, HasDecorators {
+public class OutputDeclarationImpl extends AbstractArmTreeImpl implements OutputDeclaration, HasDecorators, HasKeyword {
```

Comment:
`OutputDeclaration` should extend `HasDecorators` and `HasKeyword`

Reply:
Same here. Should the json variant return `null` for the overridden methods?

Reply:
Result: Leave at is.